packages/config-utils/package.json

```diff
@@ -17,9 +17,6 @@
     },
     "homepage": "https://github.com/RedHatInsights/frontend-components#readme",
     "peerDependencies": {
-        "jws": "^4.0.0",
-        "glob": "^7.1.7",
-        "nodesi": "^1.15.0",
-        "webpack": ">=5.0.0"
+        "webpack": "^5.0.0"
```

Comment:
We can't remove these dependencies. Applications that are using the webpack proxy (and there are quite a few of them now) will stop working if they update their config.

Reply:
These are peer dependencies in the config-utils package (aka nothing gets installed). The actual dependencies are in the config package and I've added them there (glob was missing for example).

The peer dependency list should technically be a lot longer, it's just going to be annoying to keep the packages and their versions in sync with the config-utils package. I'll add them like you suggest.

Reply:
You are right. I've thought you have changed "dependencies" not "peerDependencies". Eyes don't work. :facepalm: